democracy_club/urls.py

```diff
@@ -66,8 +66,8 @@
         name="jobs",
     ),
     path(
-        "about/impact/",
-        TemplateView.as_view(template_name="about/impact.html"),
+        "impact/",
```

Comment:
As this is [indexed by Google](https://www.google.com/search?q=inurl%3Aimpact+site%3Ademocracyclub.org.uk&hl=en&ei=sKhFY86zJInRgQa06aiQAg&ved=0ahUKEwjOy4LK2tj6AhWJaMAKHbQ0CiIQ4dUDCA4&uact=5&oq=inurl%3Aimpact+site%3Ademocracyclub.org.uk&gs_lcp=Cgdnd3Mtd2l6EANKBAhBGAFKBAhGGABQ7AdY7Adg5ghoAXAAeACAATeIATeSAQExmAEAoAEBwAEB&sclient=gws-wiz), can you set up a `RedirectView` please?